website/source/docs/commands/kv/delete.html.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: "docs"
+page_title: "kv delete - Command"
+sidebar_title: "<code>delete</code>"
+sidebar_current: "docs-commands-kv-delete"
+description: |-
+  The "kv delete" command disables an secrets engine at a given PATH. The
+  argument corresponds to the enabled PATH of the engine, not the TYPE! All
+  secrets created by this engine are revoked and its Vault data is removed.
+---
+
+# kv delete
+
+The `kv delete` command deletes the data for the provided path in
+the key/value secrets engine. If using K/V Version 2, its versioned data will
+not be fully removed, but marked as deleted and will no longer be returned in
+normal get requests.
+
+## Examples
+
+Delete the latest version of the key "creds":
+
+```text
+$ vault kv delete secret/creds
+Success! Data deleted (if it existed) at: secret/creds
+```
+
+**[K/V Version 2]** Delete version 3 of key "creds":
```

Comment:
```suggestion
**[K/V Version 2]** Delete version 11 of key "creds":
```

Reply:
Using version 3 in the example is a little confusing to me because it makes me think of KV V2 and then I get confused about whether we're talking about the secret engine or what. Using something that's a further departure from v2 makes it less confusing for me.